ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -119,7 +119,7 @@ if [ $AUTOCNF = yes ]
 then
 	param set SYS_AUTOSTART $REQUESTED_AUTOSTART
 
-	param set BAT_N_CELLS 3
+	param set BAT_N_CELLS 4
```

Comment:
Why?

Reply:
1. The `BatterySimulator` is hardcoding this value now so it's what we're already using in SITL atm: https://github.com/PX4/Firmware/blob/4088c2581f38333a306d3cd55aa66b617a51eb93/src/modules/simulator/battery_simulator/BatterySimulator.hpp#L88
1. IMHO 4S is far more common than 3S these days which was not the case when this was originally set up.
1. It doesn't matter, the simulated battery generates a voltage that considers the cell count so voltages "just look more familiar" 🤷‍♂️